packages/@ama-sdk/showcase-sdk/package.json

```diff
@@ -77,6 +74,7 @@
     "@nx/jest": "~16.10.0",
     "@o3r/eslint-config-otter": "workspace:^",
     "@o3r/eslint-plugin": "workspace:^",
+    "@o3r/workspace": "workspace:^",
```

Comment:
You should not have this deep in an app.
Why do you need it?
If it is for the set-version, do you need a set-version here instead of the workspace level only?

Reply:
When we generate an sdk we generate it with the script `set:version` it may not be needed if we are in a library but only in standalone mode. We can open an issue to fixed that point in another PR

Reply:
Agree for the task to update the generators. But this fix can be applied to our SDK in this PR

Reply:
I did it for our SDK

Reply:
https://github.com/AmadeusITGroup/otter/issues/998